### DISCUSSION TITLE
Se noen form for beskrivelse av posisjon til bysykkelstopp

### DISCUSSION THREAD

**Discussion by wfeiring** [entur/tavla#318] [Ideas]
Synes ofte det kan være vanskelig å vite hvor bysykkelstopp ligger kun basert på navnet. Hadde vært utrolig deilig med en liten beskrivelse av hvor det ligger!

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
